src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -67,15 +80,46 @@ const defaultProps = {
     iouReport: {},
     reportActions: {},
     isHovered: false,
+    session: {
+        email: null,
+    },
 };
 
 const MoneyRequestAction = (props) => {
     const hasMultipleParticipants = lodashGet(props.chatReport, 'participants', []).length > 1;
     const onIOUPreviewPressed = () => {
-        if (hasMultipleParticipants) {
+        if (lodashGet(props.action, 'originalMessage.type', '') === CONST.IOU.REPORT_ACTION_TYPE.SPLIT && hasMultipleParticipants) {
             Navigation.navigate(ROUTES.getReportParticipantsRoute(props.chatReportID));
+            return;
+        }
+
+        // If the childReportID is not present, we need to create a new thread
+        const childReportID = lodashGet(props.action, 'childReportID', '0');
+        if (childReportID === '0') {
+            const participants = _.uniq([props.session.email, props.action.actorEmail]);
+            const formattedUserLogins = _.map(participants, (login) => OptionsListUtils.addSMSDomainIfPhoneNumber(login).toLowerCase());
+            const thread = ReportUtils.buildOptimisticChatReport(
+                formattedUserLogins,
+                props.translate('iou.threadReportName', {
+                    formattedAmount: CurrencyUtils.convertToDisplayString(lodashGet(props.action, 'originalMessage.amount', 0), lodashGet(props.action, 'originalMessage.currency', '')),
+                    comment: props.action.originalMessage.comment,
+                }),
+                '',
+                CONST.POLICY.OWNER_EMAIL_FAKE,
+                CONST.POLICY.OWNER_EMAIL_FAKE,
+                false,
+                '',
+                undefined,
+                CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS,
+                props.action.reportActionID,
+                props.chatReportID, // Needs to be changed to iouReportID
+            );
```

Comment:
Eventually we should move this to reportUtils but now for urgency just leaving this pragmatically here, I hope that works

Reply:
Yea, we should create an utils method. Seems ok to leave it for now given that this is blocking other issues